packages/core/integration-tests/test/webext.js

```diff
@@ -0,0 +1,39 @@
+import path from 'path';
+import {bundle, assertBundles} from '@parcel/test-utils';
+
+describe('webext', function() {
+  it('should resolve a full webext bundle', async function() {
+    let b = await bundle(
+      path.join(__dirname, '/integration/webext/manifest.json'),
+    );
+    assertBundles(b, [
+      {
+        name: 'tmp.aff',
+        assets: ['tmp.aff'],
+      },
+      {
+        name: 'tmp.dic',
+        assets: ['tmp.dic'],
+      },
+      {
+        name: 'messages.json',
+        assets: ['messages.json'],
+      },
+      {
+        name: 'manifest.json',
+        assets: ['manifest.json'],
+      },
+      // next three are implemented to have the same name, but theoretically
+      {assets: ['a.txt']},
+      {assets: ['b.txt']},
+      {assets: ['foo.png']},
+      {assets: ['popup.html']},
+      {assets: ['devtools.html']},
+      {assets: ['content.js']},
+      {assets: ['content.css']},
+      {assets: ['background.js']},
+    ]);
+  });
+  // TODO: Test error-checking
+  // TODO: Test certain types of plugins fully rather than just one with everything
```

Comment:
1. Yes, tests for errors would indeed be great
2. I think most of the settings in the manifest don't influence each other, so I think this isn't really necessary...